CONTRIBUTING.md

```diff
@@ -149,6 +149,16 @@ git push
 
 **IMPORTANT**: By submitting a patch, you agree to the [License](#license).
 
+### Privacy Notice
+
+We would like to make you aware that contributing to OpenTTD via git will permanently store the name and email address you provide as well as the actual commit and the time and date you made it inside git's version history.
+
+This is inevitable, because it is a main feature git. If you are concerned about your privacy, we strongly recommend to use we strongly recommend to use "Anonymous &lt;anonymous@openttd.org&gt;" as the git commit author. We might refuse anonymous contributions if malicious intent is suspected.
```

Comment:
Repeated phrase "we strongly recommend to use"

Reply:
It should read for proper English "This is a main feature *of* git"